lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -118,18 +117,43 @@ class HelpPanel extends React.Component {
           </Box>
         </Box>
         <Spacing bottom={10} />
-        <Box centered displayInlineBlock fullWidth>
-          <Box displayInlineBlock fullWidth>
-            {helpItems && (
-              <For each="item" index="i" of={helpItems}>
+        <Box centered displayInlineBlock>
+          {helpItems && (
+            <For each="item" index="i" of={helpItems}>
+              <Box displayInlineBlock fullWidth styledHeight={"70px"}>
                 <HelpPanelItem
                   key={i}
                   terria={this.props.terria}
                   viewState={this.props.viewState}
                   content={item}
                 />
-              </For>
-            )}
+              </Box>
+            </For>
+          )}
+          <Box styledHeight={"64px"}>
+            <Box styledMargin={"auto 25px"}>
+              <Text semiBold extraLarge textDark uppercase>
+                <a
+                  href="//terriajs.gitbook.io/terriajs/"
```

Comment:
I think this should always be `https` and open in a new window perhaps, to avoid user loosing any datasets he has loaded?